fetch.bs

```diff
@@ -3492,6 +3832,23 @@ steps:
  <li><p>If <var>connection</var> is failure, return a
  <a>network error</a>.
 
+
+  <li><p>If the user agent supports <a for=/>Token Binding</a>, and
+  <var>request</var>'s <a for=request>use-token-binding flag</a> is set,
```

Comment:
I think you can drop the `use-token-binding flag` entirely, and just rely on the credentials flag and "user agent is configured to block cookies" here.